README.md

```diff
@@ -28,18 +28,22 @@ If using Kerberos for authentication, also follow these guides:
 Configure The Plugin
 ====
 
-This plugin provides a NodeExecutor called `overthere-winrm`, which you can set as on your node defintion:
+This plugin provides a NodeExecutor and a FileCopier called `overthere-winrm`, which you can set as on your node definition:
 
 	<node name="winNode" node-executor="overthere-winrm" .../>
 
-Or set as the default NodeExecutor for your project/framework properties file, with `service.NodeExecutor.default.provider=overthere-winrm`.
+and/or
+
+	<node name="winNode" file-copier="overthere-winrm" .../>
+
+Or set as the default NodeExecutor for your project/framework properties file, with `service.NodeExecutor.default.provider=overthere-winrm` and `service.FileCopier.default.provider=overthere-winrm`.
 
 These Node attributes are used to connect to the remote host:
 
 * `username` - Remote username. If using Kerberos, see [Using Kerberos Authentication](#using-kerberos-authentication)
 * `hostname` - Remote host. Can include "host:port" to specify port number other than the default 5985/5986 (http/https).
 
-Password authentcation can be performed in one of two ways:
```

Comment:
This change is unrelated. If you absolutely want to push it, move it to another patch (eg "README: fix typos").

Reply:
Hagrid